stream.go

```diff
@@ -132,12 +132,10 @@ func (s *Stream) WriteSysExBytes(when Timestamp, msg []byte) error {
 // the output stream. The string must only consist of hex digits (0-9A-F) and optional spaces. This
 // function is case-insenstive.
 func (s *Stream) WriteSysEx(when Timestamp, msg string) error {
-	buf, err := hex.DecodeString(strings.Replace(msg, " ", "", -1))
```

Comment:
Aha! I just noticed I was calling WriteSysEx() from my program when I should have been calling WriteSysExBytes(). This can most likely get rolled back, sorry. I am going to comb through everything to make sure I am not stepping on anything else before I send up another commit without the reimplementation. Cool?